includes/admin/upgrades/v3/upgrade-actions.php

```diff
@@ -60,10 +64,10 @@ function edd_process_v3_upgrade() {
 		wp_die( -1, 403, array( 'response' => 403 ) );
 	}
 
-	//$was_processed       = $export->process_step();                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    // @todo remove
-	$was_processed       = false; // @todo remove                                                                                                                                                                                                                                                                                                                                                                                                       // @todo remove
-	$percentage_complete = $export->get_percentage_complete();
-	$percentage_complete = 100; // @todo remove
+	$was_processed       = $export->process_step();
+	//$was_processed       = false; // @todo remove
+	$percentage_complete = round( $export->get_percentage_complete(), 2 );
```

Comment:
This is resulting in percentages like `3.88%` ... is that intentional?

Reply:
Are you referring to the two decimal places? Or what, exactly?

Reply:
Yes, sorry--I guess I was expecting all whole numbers for the percentages. So `4%` instead of `3.88%`.

Reply:
I think we need the decimals given that some migrations can be really huge. For example, on my test site when I delete legacy data, I'm below `0%` for quite a while. The decimals help reassure that it's actually processing.